server/api/controllers/broker.js

```diff
@@ -6,6 +6,46 @@ const { publisher } = require('../../db/pubsub');
 const toolsLib = require('hollaex-tools-lib');
 const { errorMessageConverter } = require('../../utils/conversion');
 
+const getBrokerQuote = (req, res) => {
+	loggerBroker.verbose(
+		req.uuid,
+		'controllers/broker/getBrokerQuote get',
+		req.auth
+	);
+
+	const user_id = req.auth.sub.id;
+	const {
+		symbol,
+		side,
+		size,
+		exchange_name,
```

Comment:
Why do we need exchange_name spread and multiplier here @fetok12 ? The formula should already include this. There is no need for a user to send these to get a quote.

Reply:
@abeikverdi, I've corrected the logic: https://github.com/bitholla/hollaex-kit/blob/1fd18d3993a7d95fff81c8c1815c1d70101e009b/server/utils/hollaex-tools-lib/tools/broker.js